bin-dir/system-test-script.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+cd ..
+
+echo "starting collection roller"
+bin/collection-roller &> /dev/null &
```

Comment:
Did you find a way to kill the processes without storing the pids?

Reply:
No we didn't found without storing PID's